chore: [#162] refresh data using HTTP Archive 2022-01-01

christobill opened the pull request:
See linked issue

patrickhulce commented:
Shoot, I'm so sorry @christobill I thought I pushed my automated updates to a branch but pushed straight to master 😞 Happy to give you credit here and add the origin-scripting copy! Apologies for the churn! Hopefully made it much easier for future updates though :D

christobill commented:
Ahah no worries @patrickhulce :smile: 

I rebased my PR, to have only the `data/2022-01-01-origin-scripting.json` as a diff

I also added a commit to update the `third-party-web` dependency in `/www` to `^13.0.0` so https://www.thirdpartyweb.today/ is based on the latest data!
